CPP Headers are not being installed

The header files are not being installed, so if the user runs `catkin build --install`, other libraries will fail to build when including the headers.

```shell
make[2]: *** [CMakeFiles/moveit_benchmark_suite_core.dir/build.make:128: CMakeFiles/moveit_benchmark_suite_core.dir/src/geometry.cpp.o] Error 1
In file included from /home/yuri/catkin_ws/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/io.h:50,
                 from /home/yuri/catkin_ws/src/moveit_benchmark_suite/core/src/io.cpp:20:
/home/yuri/catkin_ws/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/serialization/ryml.h:41:10: fatal error: moveit_serialization/ryml/conversions.h: No such file or directory
   41 | #include <moveit_serialization/ryml/conversions.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/moveit_benchmark_suite_core.dir/build.make:115: CMakeFiles/moveit_benchmark_suite_core.dir/src/io.cpp.o] Error 1
In file included from /home/yuri/catkin_ws/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/robot.h:47,
                 from /home/yuri/catkin_ws/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/planning.h:47,
                 from /home/yuri/catkin_ws/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/resource_builder.h:46,
                 from /home/yuri/catkin_ws/src/moveit_benchmark_suite/core/src/resource_builder.cpp:1:
/home/yuri/catkin_ws/src/moveit_benchmark_suite/core/include/moveit_benchmark_suite/serialization/ryml.h:41:10: fatal error: moveit_serialization/ryml/conversions.h: No such file or directory
   41 | #include <moveit_serialization/ryml/conversions.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/moveit_benchmark_suite_core.dir/build.make:63: CMakeFiles/moveit_benchmark_suite_core.dir/src/resource_builder.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2199: CMakeFiles/moveit_benchmark_suite_core.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
cd /home/yuri/catkin_ws/build/moveit_benchmark_suite_core; catkin build --get-env moveit_benchmark_suite_core | catkin env -si  /usr/bin/make --jobserver-auth=3,4; cd -

...............................................................................
Failed << moveit_benchmark_suite_core:make                                [ Exited with code 2 ]
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -129,8 +129,6 @@ target_link_libraries(ryml-quickstart ${catkin_LIBRARIES} ryml::ryml)
 ## Mark cpp header files for installation
 install(DIRECTORY include/${PROJECT_NAME}/
   DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
-  FILES_MATCHING PATTERN "*.h"
-  PATTERN ".svn" EXCLUDE
 )
 
 install(TARGETS

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -126,6 +126,13 @@ add_library(${PROJECT_NAME}
 add_executable(ryml-quickstart ext/ryml/samples/quickstart.cpp)
 target_link_libraries(ryml-quickstart ${catkin_LIBRARIES} ryml::ryml)
 
+## Mark cpp header files for installation
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+  FILES_MATCHING PATTERN "*.h"
+  PATTERN ".svn" EXCLUDE
+)
+
 install(TARGETS
     ${PROJECT_NAME}
     ryml-quickstart

